internal/cache/app_type.go

```diff
@@ -0,0 +1,28 @@
+package cache
+
+import (
+	"context"
+	"sync"
+)
+
+var (
+	appTypeId      string
+	appTypeIdMutex sync.Mutex
+)
+
+func FindAppTypeId(ctx context.Context) (string, bool) {
+	appTypeIdMutex.Lock()
+	defer appTypeIdMutex.Unlock()
+
+	if appTypeId == "" {
+		return "", false
+	}
+	return appTypeId, true
```

Comment:
```suggestion
	return appTypeId, appTypeId != ""
```
Or is this for readability?

Reply:
Sure, rebased.